Frontend/auction-app/src/Components/FormField/GenericField.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+const GenericField = ({ type, id, label, className, placeholder, onChange, name }) => {
```

Comment:
you can use prop deconstruction here to avoid writing this 2 times.
```
({label, ...props }) => { return (
     ...
    <label>{label}</label>
    <input {...props} />
     ...
 ) }
```